src/ui-timespan-picker/ui-timespan-picker.js

```diff
@@ -41,10 +41,6 @@
           model: '='
         },
         compile: function( tElement, tAttrs ) {
-          // Fix ngModel for isolate scope
-          tAttrs.$set( 'model', tAttrs.ngModel, false );
```

Comment:
This caused a problem when angular was upgraded, can you take a look at wether this is still relevant @AButler?